dashboards/resources/variables/cluster.libsonnet

```diff
@@ -1,10 +1,15 @@
+local g = import 'github.com/grafana/grafonnet/gen/grafonnet-latest/main.libsonnet';
+local var = g.dashboard.variable;
 local common = import './common.libsonnet';
 
 {
   cluster(config)::
     local datasource = common.datasource(config);
+    local clusterVar =
+      common.cluster(config, datasource, 'up{%(kubeStateMetricsSelector)s}')
+      + var.query.generalOptions.showOnDashboard.withLabelAndValue();
```

Comment:
Can this go on the common variable definition instead? Otherwise this will have to be copy/pasted every time it's used.